### PR TITLE
DOCS-10220-create-and-implement-nav-partials

### PR DESCRIPTION
**Pull request by sarathecoubian**
Also sign in -> log in

### CODE REVIEW

File: modules/ROOT/pages/_partials/include-nav-steps-no-permissions.adoc [hunk 1]
```diff
@@ -0,0 +1,2 @@
+. Log in to Anypoint Platform.
+. In the navigation bar or the main Anypoint Platform screen, click *Access Management*.
```
Comment (hannanelson) [inline modules/ROOT/pages/_partials/include-nav-steps-no-permissions.adoc]: screen -> page?

Comment (hannanelson) [reply modules/ROOT/pages/_partials/include-nav-steps-no-permissions.adoc]: (global)

File: modules/ROOT/pages/_partials/include-nav-steps-to-profile.adoc [hunk 2]
```diff
@@ -0,0 +1,2 @@
+. Log in to Anypoint Platform.
+. Navigate to your xref:managing-your-account.adoc#update-user-information[user profile].
```
Comment (hannanelson) [inline modules/ROOT/pages/_partials/include-nav-steps-to-profile.adoc]: Kim would frown on inline link. (me too :) )

File: modules/ROOT/pages/_partials/include-nav-steps-no-permissions.adoc (hunk 1 above)
Comment (hannanelson) [reply modules/ROOT/pages/_partials/include-nav-steps-no-permissions.adoc]: If you make each step a partial, you have more flexibility.

File: modules/ROOT/pages/audit-logging.adoc [hunk 3]
```diff
@@ -15,10 +15,15 @@ Users belonging to the Organization Administrator role or the Audit Log Viewer r
...
-. Sign into Anypoint Platform and click *Access Management*. +
-You can also press the `Escape` key on your keyboard to view the Anypoint Platform navigation menu.
-. From the Access Management menu on the left, select *Audit Log*. +
-The Audit logs page displays the logs.
+include::partial$include-nav-steps-org-no-permissions.adoc[]
+
+. In the Access Management navigation menu, click *Audit Logs*. +
+The *Audit Logs* page displays the logs. You can:
+
+* Download audit logs
```
Comment (hannanelson) [inline modules/ROOT/pages/audit-logging.adoc]: These are sentences (imperative) so full stop.

File: modules/ROOT/pages/business-groups.adoc [hunk 4]
```diff
@@ -83,7 +83,7 @@ image::switch-suborg.png[switch+suborg]
 
 [NOTE]
 --
-When you sign into Anypoint Platform, you are returned to the business group where you were last active when you signed out of Anypoint Platform.
+When you log in to Anypoint Platform, you are returned to the business group where you were last active when you logged out of Anypoint Platform.
```
Comment (hannanelson) [inline modules/ROOT/pages/business-groups.adoc]:
```suggestion
When you log in to Anypoint Platform, you return to the business group where you were last active when you logged out of Anypoint Platform.
```

File: modules/ROOT/pages/conf-openid-connect-task.adoc [hunk 5]
```diff
@@ -29,9 +29,10 @@ If you already configured Anypoint Platform as a client application in your iden
 
 == To Configure OpenID Connect Dynamically
 
-. Log into the master Organization in Anypoint Platform as Administrator.
-. In Anypoint Platform, click Access Management > External Identity.
-. From Identity Management, select OpenID Connect.
+include::partial$include-nav-steps-org-admin-bg.adoc[]
+
+. In the Access Management navigation menu, click *External Identity*.
+. In the *Identity Management* page, select OpenID Connect.
```
Comment (hannanelson) [inline modules/ROOT/pages/conf-openid-connect-task.adoc]:
```suggestion
. In the *Identity Management* page, select *OpenID Connect*.
```

File: modules/ROOT/pages/environments.adoc [hunk 6]
```diff
@@ -77,9 +78,9 @@ If your organization administrator provides multiple xref:managing-api-clients.a
 
 To associate a client provider with an existing environment:
 
-. Log in to Anypoint Platform using an account that has the organization administrator role.
-. Navigate to *Access Management*.
-. In the menu on the left, click *Environments*.
+include::partial$include-nav-steps-org-admin.adoc[]
+
+. In the Access Management navigation menu, click *Environments*.
```
Comment (hannanelson) [inline modules/ROOT/pages/environments.adoc:83]: You can create a partial with this, too :)

File: modules/ROOT/pages/_partials/include-nav-steps-no-permissions.adoc (hunk 1 above)
Comment (sarathecoubian) [reply modules/ROOT/pages/_partials/include-nav-steps-no-permissions.adoc]:
```suggestion
. In the navigation bar or the main Anypoint Platform page, click *Access Management*.
```

File: modules/ROOT/pages/_partials/include-nav-steps-org-admin-bg.adoc [hunk 7]
```diff
@@ -0,0 +1,3 @@
+. Log in to Anypoint Platform using an account that has the Organization Administrator permission.
+. In the navigation bar or the main Anypoint Platform screen, click *Access Management*.
```
Comment (sarathecoubian) [inline modules/ROOT/pages/_partials/include-nav-steps-org-admin-bg.adoc]:
```suggestion
. In the navigation bar or the main Anypoint Platform page, click *Access Management*.
```

File: modules/ROOT/pages/_partials/include-nav-steps-to-profile.adoc (hunk 2 above)
Comment (sarathecoubian) [reply modules/ROOT/pages/_partials/include-nav-steps-to-profile.adoc]:
```suggestion
. Navigate to your user profile.
```

File: modules/ROOT/pages/_partials/include-nav-steps-org-admin.adoc [hunk 8]
```diff
@@ -0,0 +1,2 @@
+. Log in to Anypoint Platform using an account that has the Organization Administrator permission.
+. In the navigation bar or the main Anypoint Platform screen, click *Access Management*.
```
Comment (sarathecoubian) [inline modules/ROOT/pages/_partials/include-nav-steps-org-admin.adoc]:
```suggestion
. In the navigation bar or the main Anypoint Platform page, click *Access Management*.
```

File: modules/ROOT/pages/audit-logging.adoc [hunk 9]
```diff
@@ -15,10 +15,15 @@ Users belonging to the Organization Administrator role or the Audit Log Viewer r
...
+* Set a time period for audit logs to display
+* Filter audit logs by product, type, and actions
+* Search audit logs by environment, object, and user
```
Comment (sarathecoubian) [inline modules/ROOT/pages/audit-logging.adoc]:
```suggestion
* Download audit logs.
* Set a time period for audit logs to display.
* Filter audit logs by product, type, and actions.
* Search audit logs by environment, object, and user.
```

File: modules/ROOT/pages/environments.adoc (hunk 6 above)
Comment (sarathecoubian) [reply modules/ROOT/pages/environments.adoc:83]: I could. I might revisit these nav substeps with Teams.

File: modules/ROOT/pages/multi-factor-authentication.adoc [hunk 10]
```diff
@@ -40,13 +40,13 @@ For enhanced security, you can require all users in your organization to use mul
...
+To prevent service disruptions, identify any accounts (_service accounts_) that are used to programmatically call Anypoint Platform. You must either configure these service account calls by using the xref:connected-apps-overview.adoc[Connected Apps] feature or add these accounts to the *Exempt Accounts* section when configuring multi-factor authentication.
 
 === Enable Multi-Factor Authentication for All User Accounts
 
-. Log in to Anypoint Platform using an account that has the Organization Administrator permission.
-. Navigate to *Access Management*.
-. In the navigation menu, click *Multi-factor auth*.
+include::partial$include-nav-steps-org-admin-bg.adoc[]
+
+. In the Access Management navigation menu, click *Multi-factor auth*.
```
Comment (hannanelson) [inline modules/ROOT/pages/multi-factor-authentication.adoc]: It's lowercase here?

File: modules/ROOT/pages/multi-factor-authentication.adoc [hunk 11]
```diff
@@ -60,9 +60,9 @@ It is best to use the xref:connected-apps-overview.adoc[Connected Apps] feature
 
 To exempt individual accounts from multi-factor authentication:
 
-. Log in to Anypoint Platform using an account that has the Organization Administrator permission.
-. Navigate to *Access Management*.
-. In the navigation menu, click *Multi-factor Auth*.
+include::partial$include-nav-steps-org-admin-bg.adoc[]
+
+. In the Access Management navigation menu, click *Multi-factor Auth*.
```
Comment (hannanelson) [inline modules/ROOT/pages/multi-factor-authentication.adoc:65]: but not here? (auth, I mean)

File: modules/ROOT/pages/organization.adoc [hunk 12]
```diff
@@ -43,14 +43,14 @@ Changing the name or domain name of an organization changes the deep links to an
...
 
-. Sign into your Anypoint Platform account as a user with the *Organization Administrators* role.
-. Select the *Access Management* option from the main page, or click the menu button in the top left corner.
+include::partial$include-nav-steps-org-admin.adoc[]
+
 +
 image::anypoint-platform-nav-menu.png[]
 +
-The *Organization* page is displayed, where you can view organizations and business groups and add new business groups (if enabled). 
+The *Organization* page is appears, where you can view organizations and business groups and add new business groups (if enabled).
```
Comment (hannanelson) [inline modules/ROOT/pages/organization.adoc]:
```suggestion
The *Organization* page appears, where you can view organizations and business groups and add new business groups (if enabled).
```

File: modules/ROOT/pages/roles.adoc [hunk 13]
```diff
@@ -184,7 +190,7 @@ After creating a custom role, you can assign permissions to the role. If these e
 * Secrets Manger
 * Tokenization
 +
-Depending on your permissions you may not see all of these options.
+Depending on your permissions, you may not see all of these options.
```
Comment (hannanelson) [inline modules/ROOT/pages/roles.adoc]:
```suggestion
Depending on your permissions, you might not see all of these options.
```

Comment (hannanelson) [reply modules/ROOT/pages/roles.adoc]: may is for permission; might is for possibility

File: modules/ROOT/pages/test-external-identity-task.adoc [hunk 14]
```diff
@@ -5,20 +5,22 @@ endif::[]
...
 
-. Log into your Anypoint Platform administrator account, and click Access Management > Organization.
-. In Organization, click the name of an organization listed in the Name column.
+include::partial$include-nav-steps-org-admin.adoc[]
+
+. In the Access Management navigation menu, click *Organization*.
+. In the *Organization* page, click the name of an organization listed in the Name column.
 +
-Organization Info appears.
+The *Organization Info* window appears.
```
Comment (hannanelson) [inline modules/ROOT/pages/test-external-identity-task.adoc:13]: page?

File: modules/ROOT/pages/troubleshoot-access-management.adoc [hunk 15]
```diff
@@ -3,18 +3,18 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-These are some tips about what to do if you encounter these issues. 
+These are some tips about what to do if you encounter these issues.
```
Comment (hannanelson) [inline modules/ROOT/pages/troubleshoot-access-management.adoc]:
```suggestion
These tips provide information on what to do if you encounter issues.
```

Comment (hannanelson) [reply modules/ROOT/pages/troubleshoot-access-management.adoc]: These/these
The first These needs something to modify...

File: modules/ROOT/pages/troubleshoot-access-management.adoc [hunk 16]
```diff
@@ -3,18 +3,18 @@ ifndef::env-site,env-github[]
...
 
 == Problems with Accessing Anypoint Platform
 
 === Sign in Credentials are not recognized
```
Comment (hannanelson) [inline modules/ROOT/pages/troubleshoot-access-management.adoc]: Are these headings actual errors? If they are not, then use Title Case.

File: modules/ROOT/pages/troubleshoot-access-management.adoc [hunk 17]
```diff
@@ -3,18 +3,18 @@ ifndef::env-site,env-github[]
...
+If your credentials aren't recognized when you try to log in to Anypoint Platform, try the following:
 
-* Check that you are using the correct URL to sign into your organization. +
+* Check that you are using the correct URL to log in to your organization. +
 If your organization uses an external identity management system, your login URL is *anypoint.mulesoft.com/accounts/login/_yourOrganizationDomain_*.
 * Verify that you are using the correct username and password for the account.
-* If your organization isn't using an external identity management system and your sign in credentials aren't working, reset your password or ask your Organization Administrator if your account is still active.
+* If your organization isn't using an external identity management system and your login credentials aren't working, reset your password or ask your Organization Administrator if your account is still active.
 
 === Access Management invitation link returns an error
```
Comment (hannanelson) [inline modules/ROOT/pages/troubleshoot-access-management.adoc]: Title Case

Comment (hannanelson) [reply modules/ROOT/pages/troubleshoot-access-management.adoc]: global

File: modules/ROOT/pages/troubleshooting-anypoint-platform-access.adoc [hunk 18]
```diff
@@ -37,15 +37,15 @@ Please create a https://help.mulesoft.com[MuleSoft support case] for account del
...
+If you have one or more existing MuleSoft community accounts tied to the email address at which you received your invitation, MuleSoft prompts you to use your existing account to log in and accept the invitation. You can also choose to create a new account by following the *Sign Up* link at the bottom of the form.
 
 === How can I use a different email address?
 
 Accept the invitation using the email where it was sent. After you've accepted the invitation, visit your profile to change your email address.
 
 === Why can't I see anything?
 
-If you accept an invitation, sign in, and can't see anything on your organization's Developer Portal, one of the following may be affecting your view:
+If you accept an invitation, log in, and can't see anything on your organization's Developer Portal, one of the following may be affecting your view:
```
Comment (hannanelson) [inline modules/ROOT/pages/troubleshooting-anypoint-platform-access.adoc]: Search for may and see if it really should be "might" or "can"

File: modules/ROOT/pages/troubleshooting-anypoint-platform-access.adoc [hunk 19]
```diff
@@ -57,25 +57,25 @@ If you accept an invitation, sign in, and can't see anything on your organizatio
 To give others access to your API Version, you must be the *API Version Owner*, and they must be a member of your organization.
 
 . Navigate to your API Version Details page.
-. Click the *Permissions* tab. 
+. Click the *Permissions* tab.
 . Invite a user to access your API, if that user is a member of the same organization as you.
-. Once you have added a user, you can manage *Portal Viewer* access or *API Version Owner* access, using the drop-down in the *Permission* column. This tab also allows you to view all users who have been granted direct permissions to your API. 
+. Once you have added a user, you can manage *Portal Viewer* access or *API Version Owner* access, using the drop-down in the *Permission* column. This tab also allows you to view all users who have been granted direct permissions to your API.
```
Comment (hannanelson) [inline modules/ROOT/pages/troubleshooting-anypoint-platform-access.adoc]:
```suggestion
. Once you add a user, you can manage *Portal Viewer* access or *API Version Owner* access using the drop-down in the *Permission* column. This tab also enables you to view all users who have been granted direct permissions to your API.
```

**Review by hannanelson — DISMISSED** [review]

File: modules/ROOT/pages/multi-factor-authentication.adoc (hunk 10 above)
Comment (sarathecoubian) [reply modules/ROOT/pages/multi-factor-authentication.adoc]:
```suggestion
. In the Access Management navigation menu, click *Multi-factor Auth*.
```

File: modules/ROOT/pages/test-external-identity-task.adoc (hunk 14 above)
Comment (sarathecoubian) [reply modules/ROOT/pages/test-external-identity-task.adoc:13]: It is a popup window. Kinda strange.

File: modules/ROOT/pages/troubleshoot-access-management.adoc (hunk 17 above)
Comment (sarathecoubian) [reply modules/ROOT/pages/troubleshoot-access-management.adoc]:
```suggestion
=== Access Management Invitation Link Returns an Error
```

File: modules/ROOT/pages/troubleshoot-access-management.adoc [hunk 20]
```diff
@@ -28,14 +28,10 @@ If you receive an `There was an error while talking to CoreServices` when you tr
...
 
-=== Error received when doing a redirect 
-
-When a redirect to the OpenID Connect Provider occurs, you receive an `Invalid Scopes error`. 
-
-Cause: This can happen if the `openid`, `profile`, and `email` scopes aren't supported by the auto-registered client. 
-
-Solution: Add the `openid`, `profile`, and `email` scopes to the client manually. 
-
+=== Error received when doing a redirect
```
Comment (sarathecoubian) [inline modules/ROOT/pages/troubleshoot-access-management.adoc]:
```suggestion
=== Error Received When Doing a Redirect
```

File: modules/ROOT/pages/troubleshooting-anypoint-platform-access.adoc (hunk 18 above)
Comment (sarathecoubian) [reply modules/ROOT/pages/troubleshooting-anypoint-platform-access.adoc]:
```suggestion
If you accept an invitation, log in, and can't see anything on your organization's Developer Portal, one of the following might be affecting your view:
```

File: modules/ROOT/pages/troubleshoot-access-management.adoc (hunk 16 above)
Comment (sarathecoubian) [reply modules/ROOT/pages/troubleshoot-access-management.adoc]:
```suggestion
=== Sign in Credentials are Not Recognized
```

**Review by hannanelson — APPROVED** [review]

**Pull request merged**